src/components/Nav.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class Nav extends Component {
+  render() {
+    /* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md#rule-details

Reply:
done